nncf/common/graph/patterns.py

```diff
@@ -254,3 +261,134 @@ def merge_two_types_of_operations(first_op: Dict, second_op: Dict, label: str) -
+    SWISH_WITH_SIGMOID = Pattern('swish_with_sigmoid', backends=[BackendType.ONNX])
+    SWISH_WITH_HARDSIGMOID = Pattern('swish_with_hardsigmoid', backends=[BackendType.ONNX])
+
+    # DEVICE PATTERNS
+    HSWISH_ACTIVATION_CLAMP_MULTIPLY = Pattern('hswish_activation_clamp_multiply',
+                                               devices=[TargetDevice.CPU, TargetDevice.GPU, TargetDevice.VPU])
+    LINEAR_SCALE_SHIFT = Pattern('linear_scale_shift',
+                                 devices=[TargetDevice.CPU, TargetDevice.GPU])
+    LINEAR_BIASED_SCALE_SHIFT = Pattern('linear_biased_scale_shift',
+                                        devices=[TargetDevice.CPU, TargetDevice.GPU])
+    LINEAR_ACTIVATION_SCALE_SHIFT = Pattern('linear_activation_scale_shift',
+                                            devices=[TargetDevice.CPU, TargetDevice.GPU])
+    LINEAR_BIASED_ACTIVATION_SCALE_SHIFT = Pattern('linear_biased_activation_scale_shift',
+                                                   devices=[TargetDevice.CPU, TargetDevice.GPU])
+    LINEAR_ELEMENTWISE = Pattern('linear_elementwise',
+                                 devices=[TargetDevice.CPU, TargetDevice.GPU])
+    LINEAR_BIASED_ELEMENTWISE = Pattern('linear_biased_elementwise',
+                                        devices=[TargetDevice.CPU, TargetDevice.GPU])
+    LINEAR_ACTIVATION_ELEMENTWISE = Pattern('linear_activation_elementwise',
+                                            devices=[TargetDevice.CPU, TargetDevice.GPU])
+    LINEAR_BIASED_ACTIVATION_ELEMENTWISE = Pattern('linear_biased_activation_elementwise',
+                                                   devices=[TargetDevice.CPU, TargetDevice.GPU])
+
+    # OPERATIONS
+    LINEAR_OPERATIONS = Pattern('linear_operations', match=False)
+    BATCH_NORMALIZATION_OPERATIONS = Pattern('batch_normalization_operations', match=False)
+    ATOMIC_ACTIVATIONS_OPERATIONS = Pattern('atomic_activations_operations', match=False)
+    ARITHMETIC_OPERATIONS = Pattern('arithmetic_operations', match=False)
+
+
+    @staticmethod
+    def get_patterns_by_backend(backend: BackendType) -> List['PatternNames']:
+        output = []
+        for pattern in PatternNames:
+            pattern_backends = pattern.value.backends
+            if pattern_backends is None or backend in pattern_backends:
+                output.append(pattern)
+        return output
+    
+    @staticmethod
+    def get_patterns(backend: BackendType, device: TargetDevice) -> List['PatternNames']:
+        output = []
+        for backend_pattern in PatternNames.get_patterns_by_backend(backend):
+            pattern_devices = backend_pattern.value.devices
+            if pattern_devices is None or device in pattern_devices:
+                output.append(backend_pattern)
+        return output
+
+
+class PatternManager:
+
+    def __init__(self) -> None:
+        from nncf.experimental.openvino_native.hardware.fused_patterns import OPENVINO_HW_FUSED_PATTERNS
```

Comment:
Could you please put this inside the `_get_fused_patterns_by_backend(backend)` method?